### PR TITLE
Create Destination search popup

### PR DESCRIPTION
**Pull request by BhagyaPrasadSamarathunga**
#42 
Create component for Destination search popup. Under destination popup create separate common component for destination button. 
How it's look like
Screenshot 2024-09-28 at 16 29 46

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,9 +1,18 @@
+import DestinationPopUp from '../components/DestinationPopUp/DestinationPopUp'
```
Comment (dvnishshanka) [inline src/components/Header/Header.jsx:1]: import DestinationPopUp from '../DestinationPopUp/DestinationPopUp'

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+	const handelRegionClick = (item) => {
+	}
+
 	return (
-	<header>
-		Dummy Header
-	</header>
+		<div>
+			<header>
+				Dummy Header
```
Comment (dvnishshanka) [inline src/components/Header/Header.jsx:11]: You can replace the Dummy header with DestinationPopUp Component

File: src/components/DestinationPopUp/DestinationPopUp.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    {image:World, description:"I'm flexible"},
+    {image:UnitedStates, description:"United States"},
+    {image:Spain, description:"Spain"},
+    {image:SouthEastAsia, description:"Southeast Asia"},
+    {image:SouthAmerica, description:"South America"},
+    {image:Protugal, description:"Protugal"},
+    {image:MiddleEast, description:"Middle East"},
+    {image:Italy, description:"Italy"},
+    {image:Greece, description:"Greece"}];
+
```
Comment (dvnishshanka) [inline src/components/DestinationPopUp/DestinationPopUp.jsx:27]: I noted that the Portugal, South America and Greece are not in the airbnb site. Not sure whether it should be corrected or not.

**Review by dvnishshanka — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (BhagyaPrasadSamarathunga) [reply src/components/Header/Header.jsx]: Done

File: src/components/Header/Header.jsx (hunk 2 above)
Comment (BhagyaPrasadSamarathunga) [reply src/components/Header/Header.jsx]: Done

File: src/components/DestinationPopUp/DestinationPopUp.jsx (hunk 3 above)
Comment (BhagyaPrasadSamarathunga) [reply src/components/DestinationPopUp/DestinationPopUp.jsx:21]: Done

**Review by dvnishshanka — APPROVED** [review]

**Pull request merged**